client/src/KeyguardClient.ts

```diff
@@ -190,18 +152,16 @@ export class KeyguardClient {
         command: KeyguardCommand,
         request: T,
     ) {
-        const internalRequest = KeyguardClient.publicToInternal(request);
-        this._redirectBehavior.request(this._endpoint, command, [ internalRequest ]);
+        this._redirectBehavior.request(this._endpoint, command, [ request ]);
         // return value of redirect call is received in _onResolve()
     }
 
     private async _iframeRequest<T1 extends IFrameRequest, T2 extends IFrameResult>(
         command: KeyguardCommand,
         request?: T1,
     ): Promise<T2> {
-        const args = request ? [ KeyguardClient.publicToInternal(request) ] : [];
-        const internalResult = await this._iframeBehavior.request(this._endpoint, command, args);
-        return KeyguardClient.internalToPublic(internalResult) as T2;
+        const args = request ? [request] : [];
```

Comment:
could be inlined I think. Depeding on what `[ undefined ]` returns you could even just put `[ request ]` in the next line.